### PR TITLE
SCSS: remove some low hanging dead fruit

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- removes some old SCSS
  - as far as i can tell, this is all unused. @torkelo maybe you know better? 🙏 

**Why do we need this feature?**

- SCSS is dead to us 🔪 

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

For https://github.com/grafana/grafana/issues/64129

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by tskarhed** [comment]
Verify that the removal of these files do not affect deployments. https://github.com/grafana/hosted-grafana/pull/2291/files

**Review by tskarhed — CHANGES_REQUESTED** [review]

File: public/sass/components/_json_explorer.scss [hunk 1]
```diff
@@ -1,122 +0,0 @@
...
-  }
-  .json-formatter-undefined {
-    color: $json-explorer-undefined-color;
-  }
-  .json-formatter-function {
-    color: $json-explorer-function-color;
-  }
-  .json-formatter-date {
-    background-color: fade($json-explorer-default-color, 5%);
-  }
```
[the remark this answers is not in the document]
Comment (ashharrison90) [reply public/sass/components/_json_explorer.scss:56]: nice spot, will restore this one 👍

[the remark this answers is not in the document]
Comment (ashharrison90) [reply public/sass/components/_json_explorer.scss]: meh, it's no different to the other 100 scss files in that regard 😂

**Review by tskarhed — APPROVED** [review]
When merging this, just keep an eye out for issues with deployments. Even better if we can inform is this breaks stuff. **Edit:** I can now see that the file I linked is gone, so they have changed the process.

**Pull request merged**